<?xml version="1.0" encoding="UTF-8"?>

<launch>

	<arg name="mav_name" default="quad"/>
	<arg name="id" default="0"/>
	<arg name="quad_ns"  default="$(arg mav_name)$(arg id)"/>
	<arg name="takeoff_height" default="1.5"/>
	<arg name="max_vel" default="2"/>
	<arg name="auto_takeoff" default="true"/>
	<arg name="pose_target_topic" default="command/pose"/>
	<group ns="$(arg quad_ns)">
		<node pkg="mslquad" type="default_controller" name="default_controller" output="screen">
			<param name="controller_type" value="pose_track"/>
			<param name="quad_ns" value="$(arg quad_ns)"/>
			<param name="takeoff_height" value="$(arg takeoff_height)" />
			<param name="max_vel" value="$(arg max_vel)" />
			<param name="control_freq" value="30"/>
			<param name="slow_freq" value="10"/>
			<param name="auto_takeoff" value="$(arg auto_takeoff)"/>
			<param name="pose_target_topic" value="$(arg pose_target_topic)"/>
            <param name="/simulation" value="true"/>
		</node>
    </group>
</launch>